changelog/issue-6972.md

```diff
@@ -0,0 +1,5 @@
+audience: worker-deployers
+level: patch
+reference: issue 6972
+---
+Generic Worker now uploads task payload artifacts in parallel to decrease graceful termination time in the event of a spot termination.
```

Comment:
Is it worth mentioning the change to the insecure engine as well?

Reply:
Yes I'll add this, thanks 👍🏻